CHANGELOG.md

```diff
@@ -36,6 +36,10 @@ Due to the removal of legacy code, there are a few breaking changes in this new
 
 ### New Features
 
+* Added the `contains`, `replace`, `hasPrefix`, `hasSuffix`, `lowerFirstLetter` filters for strings
```

Comment:
`.  ` (punctuation+2 spaces) missing at end of line for correct markdown formatting